src/config/wagmi.tsx

```diff
@@ -43,7 +43,9 @@ export const wagmiConfig = createConfig({
   chains: [config.chain],
   connectors,
   transports: {
-    [celo.id]: fallback([http(config.chain.rpcUrls.default.http[0]), http(infuraRpcUrl)]),
+    [celo.id]: fallback([http(config.chain.rpcUrls.default.http[0]), http(infuraRpcUrl)], {
+      rank: true,
```

Comment:
this is to pick the currently best provider [as per documentation](https://viem.sh/docs/clients/transports/fallback.html#transport-ranking)